scripts/assemble_changelog.py

```diff
@@ -96,9 +87,9 @@ def __init__(self, input_stream):
         can be any generator returning the lines to read.
         """
         self.header = []
-        self.section_list = []
-        self.section_content = {}
-        self.add_sections(*STANDARD_SECTIONS)
+        self.section_content = OrderedDict()
+        for section in STANDARD_SECTIONS:
+            self.section_content[section] = []
```

Comment:
Note: this could be shortened to 
```
self.section_content = OrderedDict((s, []) for s in STANDARD_SECTIONS)
```
But I leave it up to you to pick/keep the version you prefer.